compiler/noirc_frontend/src/elaborator/patterns.rs

```diff
@@ -447,6 +474,43 @@ impl<'context> Elaborator<'context> {
         typ
     }
 
+    fn instantiate(
+        &mut self,
+        typ: Type,
+        bindings: TypeBindings,
+        turbofish_generics: Option<Vec<Type>>,
+        function_generic_count: usize,
+        span: Span,
+        location: Location,
+    ) -> (Type, TypeBindings) {
+        match turbofish_generics {
+            Some(turbofish_generics) => {
+                if turbofish_generics.len() != function_generic_count {
+                    self.errors.push((
+                        CompilationError::TypeError(
+                            TypeCheckError::IncorrectTurbofishGenericCount {
+                                expected_count: function_generic_count,
+                                actual_count: turbofish_generics.len(),
+                                span,
+                            },
+                        ),
+                        location.file,
+                    ));
```

Comment:
Do you mind trying to fix the formatting here a bit?

Reply:
I made a separate type check error variable so that it looks like this:
```rust
                    let type_check_err = TypeCheckError::IncorrectTurbofishGenericCount {
                        expected_count: function_generic_count,
                        actual_count: turbofish_generics.len(),
                        span,
                    };
                    self.errors.push((CompilationError::TypeError(type_check_err), location.file));
```